grounding/src/test/resources/mappings/retrieval.json

```diff
@@ -0,0 +1,36 @@
+{
+  "request": {
+    "method": "GET",
+    "url": "/v2/lm/document-grounding/retrieval/dataRepositories"
+  },
+  "response": {
+    "status": 200,
+    "headers": {
+      "Content-Type": "application/json"
```

Comment:
how does the unit testing work?

Reply:
[WireMock supports file-based stubbing](https://wiremock.org/docs/stubbing/#basic-stubbing)

With that, you can define `{"reqest: ": {...}, "response": {...}}`.
(+) less verbose JSON in unit test code
(-) payload decoupled from unit test cases

I wanted to see it in comparison with our other tests. We used the approach in Cloud SDK in some cases.
I'm fine with changing from this implicit to our regular explicit format, if requested.